BUILD.md

```diff
@@ -31,6 +33,35 @@ cd seedpod_ground_risk
 pip install -r requirements.txt
 ```
 
+## Linux
+
+1. Install dependencies
+
+```bash
+sudo apt install libgeos-dev libgdal-dev proj-bin libbz2-dev
+```
+
+2. Create and activate a virtual environment, any should do. Here we use `virtualenv`:
+
+```bash
+python -m venv spgr
+source spgr
+```
+
+3. Clone the repository
+
+```commandline
+git clone https://github.com/aliaksei135/seedpod_ground_risk.git
+```
+
+4. Install requirements
+
+```commandline
+cd seedpod_ground_risk
+
+pip install -r requirements-linux.txt
```

Comment:
Not sure if this is a dumb question or not, but what is the purpose of building the wheel below? I believe you can just `pip install .` at this point to download/install all dependencies and install the project.

Reply:
You can if you are installing on the same environment. These instructions are more targeted towards packaging, particularly because the wheels are too large to upload to PyPi etc. The thinking being that you can then copy the same wheels to other machines which don't necessarily have constant network connections.

Reply:
Cool, sounds good to me!

Reply:
Normally I would agree, but we have usecases where the laptops used for the UAS ground stations do not have internet connections!